sources/Renderer/Bootstrap/FieldRenderer/BsSelectObjectFieldRenderer.php

```diff
@@ -381,7 +388,7 @@ protected function RenderHierarchicalSearch(RenderingOutput &$oOutput)
 <<<JS
 				$('#{$sHierarchicalButtonId}').off('click').on('click', function(){
 					// Creating a new modal
-					CombodoModal.OpenModal({
+					CombodoPortalToolbox.OpenModal({
```

Comment:
```suggestion
					CombodoModal.OpenModal({
```

Revert this, it is not related to that PR and this API has been deprecated.